extensions/Vendor/FRAUNHOFER_materials_pbr/README.md

```diff
@@ -0,0 +1,146 @@
+For `metallic = 1`, `base` is the specific measured reflectance value (F0).
+For `metallic = 0`, `base` represents the reflected diffuse color of the material.
+
+In this model it is not possible to specify a reflectance value for non-metals, where a reflectance value of 4% (0.04) is often used.
+The `roughness` property is related with the `glossiness` parameter in the specular-glossiness model and is defined as `roughness = 1 - glossiness`. Figure 2 shows the three components of the metal-roughness model and the rendered result.
+
+The following table lists the allowed types and ranges for the metal-roughness model:
+
+| Property | Type | Range | Default Value | Description |
+|:------------:|:----:|:-----:|:-:|:-----:|:-----------:|
+| `baseColorFactor`           | `FLOAT_VEC4` | [0, 1] for all components | [1,1,1,1] | The RGBA components of the base color (RGB) of the material. The fourth component (A) is the `opacity` of the material. |
+| `metallicFactor`            | `FLOAT`      | [0, 1]                    | 1         | The metallic-ness the material (1 for metals, 0 for non-metals). |
+| `roughnessFactor`           | `FLOAT`      | [0, 1]                    | 1         |The roughness of the material surface. |
+| `baseColorTexture`          | string       | valid texture identifier  |           |Texture with the RGBA components of the base color (RGB) of the material. If the fourth component (A) is present, it represents the `opacity` of the material. Otherwise, an `opacity` of 1 is assumed. |
+| `metallicRoughnessTexture`  | string       | valid texture identifier  |           |Texture with two (or more) components, containing the metallic-ness of the material (first component) and its roughness (second component). |
+
+The factors (baseColorFactor, metallicFactor, roughnessFactor) scale the components given in the respective textures (baseColorTexture, metallicRoughnessTexture).
+If a texture is not given, all respective texture components are assumed to have a value of 1. This is similar to the handling of factors and texture within the specular-glossiness model.
+
+
+<strong>Usage Example:</strong>
+
+```javascript
+"materials": {
+    "rough_gold": {
+        "extensions": {
+            "KHR_materials_pbr" : {
+                "technique" : "PBR_metal_roughness",
+                "values": {
+                    "baseColorFactor": [ 0.5, 0.5, 0.5, 1 ],
+                    "metallicFactor": 0.0,
+                    "roughnessFactor": 0.2
+                }
+            }
+        }
+    }
+}
+```
+
+
+## glTF Schema
+
+
+* [specular-glossiness](schema/materials_pbr_specular_glossiness.schema.json)
+* [metal-roughness](schema/materials_pbr_metal_roughness.schema.json)
+
+
+## Appendix
+
+An introduction to PBR concepts is provided in the [appendix](Appendix.md).
+
+
+## Known Implementations
```

Comment:
Would you consider contributing a stage to [gltf-pipeline](https://github.com/AnalyticalGraphicsInc/gltf-pipeline) to convert a glTF model with PBR materials using this extension to a glTF model with generated materials/techniques/shaders?

This would:
* Create a low barrier-to-entry for users to get started with this extension
* Serve as a reference implementation for those implementing the extension (JavaScript developers could probably use parts of it as is)
* Help ensure consistent implementations of this extension across engines